packages/app/src/cli/api/graphql/push_config.ts

```diff
@@ -7,7 +7,7 @@ export const PushConfig = gql`
     $applicationUrl: Url
     $redirectUrlAllowlist: [Url]
     $requestedAccessScopes: [String!]
-    $webhookApiVersion: String!
+    $webhookApiVersion: String
```

Comment:
I had to remove this requirement otherwise net new apps can't run `push` with an undefined webhookApiVersion. We can require it on the `deploy` command.

The question is, would it break anything for net new apps if this remained `undefined`? ie, can a net new app with declarative config work properly without an api version if a developer only runs `push` and doesn't run `deploy`? We automatically set it (manually and in CLI) whenever an app is created and linked to partners.

Reply:
why did we even want to make this change here now? We are also planning on storing `webhookApiVersion` as a part of the webhooks module correct? If thats the case, then im on board with doing this but I feel like this should be one of the last things we do, as we want to keep the CLI push/deploy experience pretty much the exact same right up until we release declarative webhooks.

Reply:
edit: nevermind, this is toggled by beta! I totally missed that